docs/model_developers/release.rst

```diff
@@ -1,12 +1,17 @@
 Releasing Modules
 =================
 
-Inmanta modules are versioned based on git tags. The current version is reflected in the ``module.yml`` file and in the
-commit is should be tagged in the git repository as well. To ease the use inmanta provides a command (``inmanta modules
-commit``) to modify module versions, commit to git and place the correct tag.
+.. warning::
+
+   The V2 module format is currently under development.
+
+Inmanta modules are versioned based on git tags. The current version is reflected in the ``module.yml`` file for V1 modules or
```

Comment:
This is not necessarily the case for V2 modules, is it? I assume we will continue tagging modules but it isn't strictly part of the public requirements I think.

Reply:
I think it is still best practice, so I would keep it in the docs.

Reply:
All right, I can get behind that.